docs/faq.asciidoc

```diff
@@ -0,0 +1,99 @@
+[[ecs-faq]]
+== FAQ
+
+[float]
+=== What are the benefits of using ECS?
+
+The benefits to a user adopting these fields and names in their clusters are:
+
+* **Data correlation.** Ability to easily correlate data from the same or different sources, including:
+** data from metrics, logs, and application performance management (apm) tools
+** data from the same machines/hosts
+** data from the same service
+* **Ease of recall.** Improved ability to remember commonly used field names (because there is a single set, not a set per data source)
+* **Ease of deduction.** Improved ability to deduce field names (because the field naming follows a small number of rules with few exceptions)
+* **Reuse.** Ability to re-use analysis content (searches, visualizations, dashboards, alerts, reports, and machine learning jobs) across multiple data sources
+* **Future proofing.** Ability to use any future Elastic-provided analysis content in your environment without modifications
+
+[float]
+=== What if I have fields that conflict with ECS?
+
+The
+https://www.elastic.co/guide/en/elasticsearch/reference/current/rename-processor.html[rename
+processor] can help you resolve field conflicts. For example, imagine that you
+already have a field called "user," but ECS employs `user` as an object. You can
+use the rename processor on ingest time to rename your field to the matching ECS
+field. If your field does not match ECS, you can rename your field to
+`user.value` instead.
+
+[float]
+=== What if my events have additional fields?
+
+Events may contain fields in addition to ECS fields. These fields can follow the
+ECS naming and writing rules, but this is not a requirement.
+
+[float]
+=== Why does ECS use a dot notation instead of an underline notation?
+
+There are two common key formats for ingesting data into Elasticsearch:
+
+* Dot notation: `user.firstname: Nicolas`, `user.lastname: Ruflin`
+* Underline notation: `user_firstname: Nicolas`, `user_lastname: Ruflin`
+
+For ECS we decided to use the dot notation. Here's some background on this decision.
```

Comment:
Let's reinforce the meaning of dots here at the introduction of the concept. It's well laid out below for those who will pore over each section. Would this make sense?

```suggestion
For ECS we decided to use the dot notation, representing nested objects. Here's some background on this decision.
```